packages/webapp/src/pages/Integrations/providerConfigKey/Endpoints/Show.tsx

```diff
@@ -39,12 +25,7 @@ export const EndpointsShow: React.FC<{ integration: GetIntegration['Success']['d
         const tmp: Record<string, NangoSyncConfigWithEndpoint[]> = {};
         for (const flow of data.flows) {
             for (const endpoint of flow.endpoints) {
-                const paths = endpoint.path.split('/');
-                const firstPath = paths[1];
-                if (!firstPath) {
-                    continue;
-                }
-                const groupName = allowedGroup.includes(firstPath) ? firstPath : 'others';
+                const groupName = endpoint.entity || 'others';
```

Comment:
should we have a migration query to populate the field with the first part of the path for existing endpoints in order to recreate the current behavior?

Reply:
mmh good point. We have planned a task to backfill all templates and customers, I don't think we need to go further  honestly but cc @bastienbeurier

Reply:
I think we're okay waiting for the backfill, unless @khaliqgant disagrees.

Reply:
All good with waiting for the backfill